src/sdk/model/user/user.ts

```diff
@@ -768,4 +774,58 @@ export class User {
       return json.map((it) => new VCCFailoverPlan(it));
     });
   }
+
+  /**
+   * Get the O365 Organizations for a company and user
+   * @param {string} companyId
+   * @param {string} locationId
+   * @return {Promise<Array<O365Organization>>}
+   */
+  /* istanbul ignore next: autogenerated */
+  async getO365Organizations(companyId: string, locationId?: string): Promise<Array<O365Organization>> {
```

Comment:
Can document the default for the optional locationId here and for the other methods that have it as a param too.

Reply:
@SuperConfuserUser sorry not sure what you mean here?

Reply:
Does the optional locationId have a default value that's handled by the backend or a null value like O365SharePointRestoreOptionsRequest.list? Or is it optional in some cases like I think one of the other values like folderId(or somethingElseId?) might've been if it was at the top directory? Sorry, not remembering it correctly and can't find the reference.

getO365Jobs and getO365RestoreSessions are also using an optional locationId.

Reply:
@SuperConfuserUser the optional locationId param is used as a filter on all these user resource endpoints. If no default, all locations come back on the GET requests :)